slices/tzdata.yaml

```diff
@@ -0,0 +1,300 @@
+package: tzdata
+
+slices:
+  zoneinfo:
+    essential:
+      - tzdata_africa
+      - tzdata_america
+      - tzdata_antarctica
+      - tzdata_arctic
+      - tzdata_atlantic
+      - tzdata_australia
+      - tzdata_brazil
+      - tzdata_canada
+      - tzdata_chile
+      - tzdata_etc
+      - tzdata_eurasia
+      - tzdata_indian
+      - tzdata_mexico
+      - tzdata_pacific
+      - tzdata_united-states
+      - tzdata_zoneinfo-base
```

Comment:
Oh wait, is the list in this page faulty?
https://packages.ubuntu.com/jammy/all/tzdata/filelist

I later downloaded the deb and extracted myself. Upon comparison I only get the following minimal diff instead of the late one before:
```diff
1d0
< /usr/sbin/tzconfig
3,12d1
< /usr/share/doc/tzdata/changelog.Debian.gz
< /usr/share/doc/tzdata/README.Debian
< /usr/share/zoneinfo-icu/44/be/timezoneTypes.res
< /usr/share/zoneinfo-icu/44/be/windowsZones.res
< /usr/share/zoneinfo-icu/44/be/zoneinfo64.res
< /usr/share/zoneinfo-icu/44/be/metaZones.res
< /usr/share/zoneinfo-icu/44/le/timezoneTypes.res
< /usr/share/zoneinfo-icu/44/le/windowsZones.res
< /usr/share/zoneinfo-icu/44/le/zoneinfo64.res
< /usr/share/zoneinfo-icu/44/le/metaZones.res
```

Reply:
That sounds about right. The `zoneinfo-icu` slice was intentionally not included in `zoneinfo` since it is very specific to the ICU library.